src/ztc/api/serializers/besluittype.py

```diff
@@ -27,6 +30,15 @@ class Meta:
             'catalogus': {
                 'lookup_field': 'uuid',
             },
+            'begin_geldigheid': {
+                'source': 'datum_begin_geldigheid'
+            },
+            'einde_geldigheid': {
+                'source': 'datum_einde_geldigheid'
+            },
+            'draft': {
+                'read_only': True,
```

Comment:
~~hrm, I'd say this depends on the current value of the field, no? If `draft` is `true` -> it's writable to set it to `false`, and after that it's read-only~~

nvm, I saw later on that it's implemented via a different action :+1: